backend/database.js

```diff
@@ -156,6 +156,17 @@ export default async () => {
         }
         await writeData(key, value)
         cache.set(key, value)
+      },
+      'chelonia/db/delete': async function (key: string): Promise<void> {
+        checkKey(key)
+        if (key.startsWith('_private_immutable_')) {
+          const existingValue = await readData(key)
+          if (existingValue !== undefined) {
+            throw new Error('Cannot delete already set immutable key')
+          }
```

Comment:
This check doesn't make sense, as it implies `_private_immutable` keys would be deleted if they don't exist (at which point, there's no point in deleting them).

Also, to be consistent, if any check is performed, it should be for `_private_immutable` instead of `_private_immutable_`